ios/Video/RCTVideo.m

```diff
@@ -1522,20 +1522,13 @@ - (void)usePlayerLayer
 
 - (void)setControls:(BOOL)controls
 {
-  if( _controls != controls || (!_playerLayer && !_playerViewController) )
+  if( _controls != controls || !_playerViewController )
   {
-    _controls = controls;
-    if( _controls )
-    {
-      [self removePlayerLayer];
+    if(!_playerViewController) {
       [self usePlayerViewController];
     }
-    else
-    {
-      [_playerViewController.view removeFromSuperview];
```

Comment:
Are we not remiving the view from superview anymore? this will result in a memoery leak.

Reply:
Given that we only use this controller now when viewing a video, I don't think there would be a need to ever remove it from superview?